styles.css

```diff
@@ -4,4 +4,5 @@ html {
 
 body {
   text-align: center;
-}
+  color: blue;
```

Comment:
Oooo thanks for catching my error -- you are 100% right, it is `color` not `text-color`.